Package.swift

```diff
@@ -1,151 +1,67 @@
-     }
-
-     static func from(_ content: String) -> Result<Netrc, Error> {
-         let trimmedCommentsContent = trimComments(from: content)
-         let tokens = trimmedCommentsContent
-             .trimmingCharacters(in: .whitespacesAndNewlines)
-             .components(separatedBy: .whitespacesAndNewlines)
-             .filter({ $0 != "" })
-
-         var machines: [NetrcMachine] = []
-
-         let machineTokens = tokens.split { $0 == "machine" }
-         guard tokens.contains("machine"), machineTokens.count > 0 else { return .failure(NetrcError.machineNotFound) }
-
-         for machine in machineTokens {
-             let values = Array(machine)
-             guard let name = values.first else { continue }
-             guard let login = values["login"] else { return .failure(NetrcError.missingValueForToken("login")) }
-             guard let password = values["password"] else { return .failure(NetrcError.missingValueForToken("password")) }
-             machines.append(NetrcMachine(name: name, login: login, password: password))
-         }
-
-         guard machines.count > 0 else { return .failure(NetrcError.machineNotFound) }
-         return .success(Netrc(machines: machines))
-     }
-
-     private static func trimComments(from text: String) -> String {
-         let regex = try! NSRegularExpression(pattern: "\\#[\\s\\S]*?.*$", options: .anchorsMatchLines)
-         let nsString = text as NSString
-         let range = NSRange(location: 0, length: nsString.length)
-         let matches = regex.matches(in: text, range: range)
-         var trimmedCommentsText = text
-         matches.forEach {
-             trimmedCommentsText = trimmedCommentsText
-                 .replacingOccurrences(of: nsString.substring(with: $0.range), with: "")
-         }
-         return trimmedCommentsText
-     }
- }
-
- fileprivate extension Array where Element == String {
-     subscript(_ token: String) -> String? {
-         guard let tokenIndex = firstIndex(of: token),
-             count > tokenIndex,
-             !["machine", "login", "password"].contains(self[tokenIndex + 1]) else {
-                 return nil
-         }
-         return self[tokenIndex + 1]
-     }
- }
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "MapboxMaps",
+    defaultLocalization: "en",
+    platforms: [.iOS(.v11)],
+    products: [
+        .library(
+            name: "MapboxMaps",
+            targets: ["MapboxMaps"]),
+    ],
+    dependencies: [
+        .package(name: "MapboxCommon", url: "https://github.com/mapbox/mapbox-common-ios.git", .exact("10.0.0-beta.11")),
+        .package(name: "MapboxCoreMaps", url: "https://github.com/mapbox/mapbox-core-maps-ios.git", .exact("10.0.0-beta.15")),
+        .package(name: "MapboxMobileEvents", url: "https://github.com/mapbox/mapbox-events-ios.git", .exact("0.10.8")),
```

Comment:
Does this need to be `exact`, or can it be `from:` like the Turf dependency and Cartfile?